lib/response.js

```diff
@@ -176,8 +176,10 @@ res.send = function send(body) {
   var generateETag = !this.get('ETag') && typeof etagFn === 'function'
 
   // populate Content-Length
-  var len
-  if (chunk !== undefined) {
+  // Because Content-Length and Transfer-Encoding can't be present in the response headers together,
+  // Content-Length should be added only if there is no Transfer-Encoding header
+  var len;
+  if (chunk !== undefined && !this.getHeader("transfer-encoding")) {
```

Comment:
`this.getHeader("transfer-encoding")` should be `this.get("Transfer-Encoding")`

Reply:
Makes sense.